### ISSUE TITLE
[Tacotron2-Waveglow/Pytorch] Is this code compatible with Tesla K80??

### ISSUE THREAD

**Issue by fosimoes** [NVIDIA/DeepLearningExamples#350]
I am getting the following error when running Tacotron/Waveglow on a Tesla K80:

RuntimeError: CUDA error: no kernel image is available for execution on the device

I tried to compile Pytorch from source (as suggested here: https://github.com/NVIDIA/DeepLearningExamples/issues/308)
I forced compute capability 3.7 when compiling Pytorch, which seems to be the correct value for K80.
These are the lines I added to the Dockerfile:

#Install pytorch from source
WORKDIR /workspace
RUN git clone --recursive https://github.com/pytorch/pytorch
WORKDIR /workspace/pytorch
RUN git submodule sync
RUN git submodule update --init --recursive
RUN export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
RUN TORCH_CUDA_ARCH_LIST="3.7" python setup.py install

This worked for Tacotron, but not for Waveglow training.  In this case, I still get the following error:

Traceback (most recent call last):
  File "train.py", line 538, in <module>
    main()
  File "train.py", line 456, in main
    loss.backward()
  File "/opt/conda/lib/python3.6/site-packages/torch/tensor.py", line 195, in backward
    torch.autograd.backward(self, gradient, retain_graph, create_graph)
  File "/opt/conda/lib/python3.6/site-packages/torch/autograd/__init__.py", line 99, in backward
    allow_unreachable=True)  # allow_unreachable flag
RuntimeError: CUDA error: no kernel image is available for execution on the device

Inference using pre-trained models did not work as well. I suppose pre-trained models use mixed-precision, which is not available for K80... In this case, I must run my own training from scratch in order to test inference, is that correct?

Summing up: is it worth insisting to run this code on a K80, or should I assume it requires Volta/Turing GPUs?

**Comment by GrzegorzKarchNV** [comment]
@fosimoes have you tried inference and training in fp32?
there are pretrained models in FP32 available on NGC:
https://ngc.nvidia.com/models/nvidian:swdl:tacotron2pyt_fp32
https://ngc.nvidia.com/models/nvidian:swdl:waveglowpyt_fp32

**Comment by GrzegorzKarchNV** [comment]
this is a duplicate of https://github.com/NVIDIA/DeepLearningExamples/issues/333
